### ISSUE TITLE
Nailfold capillary tortuosity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6378]
**Preferred term label:**
Nailfold capillary tortuosity
see https://pubmed.ncbi.nlm.nih.gov/25029335/
https://omim.org/entry/615934
also, periungual erythema  3/3 each

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
